repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -409,6 +409,8 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
                   case Success(newAdminToken) => requestHandler.getGroups(newAdminToken, validToken.userId)
                   case Failure(x) => Failure(IdentityAdminTokenException("Unable to reacquire admin token", x))
                 }
+              case _: InvalidTokenException if Option(keystoneV2Config.getIdentityService.getSetRackersGroups).isDefined && validToken.roles.exists(_.equalsIgnoreCase("racker")) =>
```

Comment:
If you have concerns, we may as well leave it case-insensitive. It's configurable behavior specific to Rackspace, and I assume Rackspace will never create a "racker" role AND a "Racker" role that are distinct. If so, oh man, Identity and I are going to get into a fight.